cmd/kluctl/commands/utils.go

```diff
@@ -246,6 +254,30 @@ func parseRepoOverride(s string) (ret repocache.RepoOverride, err error) {
 	if len(sp2) == 3 {
 		ret.Ref = sp2[2]
 	}
-	ret.Override = sp[1]
+	ret.Override = 	os.ExpandEnv(sp[1])
```

Comment:
Kluctl should better not do `ExpandEnv` and instead rely on the calling shell to do this properly.

Reply:
make sense if used on CLI, less if it would come as part of the config in .kluctl.yml once.
but agree - for now will revert

Reply:
I'm not sure we should have such a feature as part of the .kluctl.yml, as that would not be in the scope of what .kluctl.yml tries to achieve (defining targets).

If you want to have such options as part of your source/repo, you can always revert to tools like https://github.com/direnv/direnv which would then include the necessary environment variables to be active while you're inside the project dir. That's how we solve such things usually.